az-digital/az_quickstart
# Added bootstrap responsive embed classes to default remote video display

This ticket adds the bootstrap responsive embed classes to default remote video display

## Description
This ticket adds the bootstrap responsive embed classes to default remote video display: https://digital.arizona.edu/arizona-bootstrap/docs/2.0/utilities/embed/

Right now this has only been applied to the default display. Functionality and displays for setting sizes (small, medium, large etc.) should be addressed in a separate ticket.

## Related Issue
Fixes #629 

## How Has This Been Tested?
To test:
-Add a Page
-Add a paragraphs where you'd expect to add video
-In the CKeditor, embed media
-Choose Remote Video and enter a URL to existing video (e.g. youtube url)
-Save and save page

Video should appear full width for what screen size you are on (and be responsive if you adjust browser size)

## Types of changes


## Checklist:
